ext/mbstring/common_codepoints.txt

```diff
@@ -3,6 +3,11 @@
 0x0020	0x007E # ASCII
 0x00A1	0x00AC # Pound sign, Yen sign, copyright sign...
 0x00AE	0x00FF # Accented Latin characters
+0x0104	0x0107 # Polish
+0x0118	0x0119 # Polish
+0x0141	0x0144 # Polish
+0x015A	0x015B # Polish
+0x0179	0x017C # Polish
```

Comment:
I think this whole "part 18" request is a PHP 8.2 thingy. But this single commit is more of a bugfix, so should be separated and targeted on 8.1, please. I don't see an added test for this specific change. Shouldn't there be one?

Reply:
Could you provide a Polish string which we can use as a test case?

Reply:
Here's some Polish sentences.
Hardcore: Zażółć gęślą jaźń.
Normal: Wół poszedł spać bardzo wcześnie. A to zdanie bez ogonka.

Reply:
OK, I am adding these to our test suite. Thanks!